src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags.